### PR TITLE
Add test cases for daos & beans generation. Enjoy it !

### PR DESCRIPTION
**Pull request by guillaumeVDP**

### CODE REVIEW

File: tests/Mouf/Database/TDBM/TDBMDaoGeneratorTest.php [hunk 1]
```diff
@@ -36,25 +30,35 @@
...
-     */
 	public function testDaoGeneration() {
-        $daoFactoryClassName = "daoFactory";
-        $sourcedirectory = "Mouf/Database/TDBM/Dao";
-        $daonamespace = "Mouf/Database/TDBM/Dao/";
-        $beannamespace = "Mouf/Database/TDBM/Dao/Bean/";
+        $daoFactoryClassName = "DaoFactory";
+        // Put your right directory from your home
+        // Todo, find a good way to do it
+        $sourcedirectory = "projects/mouf/database.tdbm/tests/Mouf/Database/TDBM";
```
Comment (guillaumeVDP) [inline tests/Mouf/Database/TDBM/TDBMDaoGeneratorTest.php:37]: If you have a better way to do it, please tell me !

Comment (moufmouf) [inline tests/Mouf/Database/TDBM/TDBMDaoGeneratorTest.php:37]: Yes! :)
Create a directory /tests/generated in the package (put an empty `.gitkeep` file in it so you can commit it).
Then, refer to this directory using the **DIR** constant.

For instance:
`$sourceDirectory = __DIR__.'/../../../../../generated';` 

=> Note: count the number of ".." correctly, it is the hardest part.

**Comment by moufmouf** [comment]
Ping! @tapotigner What do we do with this PR? Is it still useful?

File: tests/Mouf/Database/TDBM/TDBMDaoGeneratorTest.php (hunk 1 above)
Comment (tapotigner) [inline tests/Mouf/Database/TDBM/TDBMDaoGeneratorTest.php]: Yes, for the moment there is no test cases for DAOs and Beans generation. For the moment, I don't have a lot of time. I will finish them after the 15th August, is it ok for you ?

**Comment by guillaumeVDP** [comment]
Yes, for the moment there is no test cases for DAOs and Beans generation. For the moment, I don't have a lot of time. I will finish them after the 15th August, is it ok for you ?

**Comment by moufmouf** [comment]
Fixed in other branch

**Pull request closed without merging**